packages/service-library/src/servicelib/observer.py

```diff
@@ -6,33 +6,58 @@
 import logging
 from collections import defaultdict
 from functools import wraps
+from typing import Callable
 
 from .utils import logged_gather
 
 log = logging.getLogger(__name__)
 
-event_registry = defaultdict(list)
+_event_registry = defaultdict(list)
```

Comment:
A though: maybe it would be a good opportunity to have a setup and move this into the application, instead of having it use a global.